src/main/java/gr8pefish/ironbackpacks/container/ContainerBackpack.java

```diff
@@ -39,6 +39,10 @@
      */
     private int blocked = -1;
     private ItemStack blockedStack = ItemStack.EMPTY;
+    /**
+     * In which slot of the player's inventory is the backpack.
```

Comment:
Should not be a javadoc comment. Neither should the existing one above, really. Just make it a standard single line comment.

Reply:
I only changed this one comment I introduced. Trying to keep it contained to the issue.